src/Umbraco.Core/Routing/PublishedRouter.cs

```diff
@@ -413,6 +413,16 @@ internal void FindPublishedContent(IPublishedRequestBuilder request)
                     _logger.LogDebug("Finder {ContentFinderType}", finder.GetType().FullName);
                     return finder.TryFindContent(request);
                 });
+
+                _profilingLogger.DebugDuration<ProfilingLogger>(
+                    //TODO make structured logging working in profillingLogger
+                    string.Format("Found? {Found} Content: {PublishedContentId}, Template: {TemplateAlias}, Domain: {Domain}, Culture: {Culture}, StatusCode: {StatusCode}",
```

Comment:
This breaks all front-end requests because it's not a valid log message. I'll fix.

```
System.FormatException: Input string was not in a correct format.
   at System.Text.ValueStringBuilder.ThrowFormatError()
   at System.Text.ValueStringBuilder.AppendFormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.FormatHelper(IFormatProvider provider, String format, ParamsArray args)
   at System.String.Format(String format, Object[] args)
   at Umbraco.Cms.Core.Routing.PublishedRouter.FindPublishedContent(IPublishedRequestBuilder request) in C:\Users\Shannon\Documents\_Projects\Umbraco\CMS\Umbraco_netcore\src\Umbraco.Core\Routing\PublishedRouter.cs:line 417
   at Umbraco.Cms.Core.Routing.PublishedRouter.FindPublishedContentAndTemplate(IPublishedRequestBuilder request) in C:\Users\Shannon\Documents\_Projects\Umbraco\CMS\Umbraco_netcore\src\Umbraco.Core\Routing\PublishedRouter.cs:line 370
   at Umbraco.Cms.Core.Routing.PublishedRouter.RouteRequestAsync(IPublishedRequestBuilder request, RouteRequestOptions options) in C:\Users\Shannon\Documents\_Projects\Umbraco\CMS\Umbraco_netcore\src\Umbraco.Core\Routing\PublishedRouter.cs:line 165
```

Reply:
I have fixed this. This shouldn't be a DebugDuration here anyways since it's not being timed (i.e. it needs to be disposed to be timed). This is just a normal log output.

I have another PR to enable structured logging with the profiling logger.